test/restDocs/test.spec.ts

```diff
@@ -1,20 +1,29 @@
-// cspell:word colordiff
-
-import assert from 'node:assert';
-import {execSync, spawnSync} from 'node:child_process';
-import {readFileSync} from 'node:fs';
+import {execSync} from 'node:child_process';
```

Comment:
Can you please change `execSync` with "rm -rf ${resultFolder}" to use fs.rm() with recursive: true?
https://nodejs.org/api/fs.html#fspromisesrmpath-options
This way we'll hopefully bring back Windows support